EasyPost.Net/Client.cs

```diff
@@ -34,59 +34,78 @@ public int RequestTimeoutMilliseconds
             set => _requestTimeoutMilliseconds = value;
         }
 
-        private string UserAgent => $"EasyPost/v2 CSharpClient/{_libraryVersion} .NET/{_dotNetVersion}";
+        private string UserAgent => $"EasyPost.Net/v2;CSharpClient/{_libraryVersion};.NET/{_dotNetVersion}";
 
         /// <summary>
-        ///     Constructor for the EasyPost client.
+        ///     Constructor for the EasyPost.Net client.
```

Comment:
This comment should remain the same.

Reply:
Resolved in [d903ec7](https://github.com/EasyPost/easypost-csharp/pull/214/commits/d903ec7476be02f732e6eb4245a85595fc72df59)